scripts/exregional_make_ics.sh

```diff
@@ -681,20 +681,140 @@ located in the following directory:
 #
 #-----------------------------------------------------------------------
 #
-# Move initial condition, surface, control, and 0-th hour lateral bound-
-# ary files to ICs_BCs directory.
+# Run large-scale blending
+#
+#-----------------------------------------------------------------------
+# NOTES:
+# * The large-scale blending is broken down into 4 major parts
+#   1) chgres_winds: This part rotates the coldstart winds from chgres to the model D-grid.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L433, and it
+#                    is equivalent to the fv3jedi tool called ColdStartWinds.
+#   2) remap_dwinds: This part vertically remaps the D-grid winds.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L3485, and it
+#                    is part of the fv3jedi tool called VertRemap.
+#   3) remap_scalar: This part vertically remaps all other variables.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L2942, and it
+#                    is the other part of the fv3jedi tool called VertRemap.
+#   4) raymond:      This is the actual blending code which uses the raymond filter. The
+#                    raymond filter is a sixth-order tangent low-pass implicit filter
+#                    and can be controlled via the cutoff length scale (Lx).
+#
+# * Currently blended fields: u, v, t, dpres, and sphum
+#     -) Blending only works with GDASENKF (netcdf)
+#
+# * Two RRFS EnKF member files are needed: fv_core and fv_tracer.
+#     -) fv_core contains u, v, t, and dpres
+#     -) fv_tracer contains sphum
+#
+# * Before we can do any blending, the coldstart files from chgres need to be
+#   processed. This includes rotating the winds and vertically remapping all the
+#   variables. The cold start file has u_w, v_w, u_s, and v_s which correspond 
+#   to the D-grid staggering.
+#     -) u_s is the D-grid south tangential wind component (m/s)
+#     -) v_s is the D-grid south face normal wind component (m/s)
+#     -) u_w is the D-grid west  face tangential wind component (m/s)
+#     -) v_w is the D-grid west  face normal wind component (m/s)
+#     -) https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/blob/bdeee64e860c5091da2d169b1f4307ad466eca2c/tools/external_ic.F90
+#     -) https://dtcenter.org/sites/default/files/events/2020/20201105-1300p-fv3-gfdl-1.pdf
+#
+
+cdate_crnt_fhr_m1=$( date --utc --date "$yyyymmdd $hh UTC - 1 hours" "+%Y%m%d%H" )
+#echo current cycle date       : $cdate_crnt_fhr
+#echo previous cycle date      : $cdate_crnt_fhr_m1
+#echo first_cycle date         : ${DATE_FIRST_CYCL}${STARTHOUR}
+#echo first blended cycle date : $FIRST_BLENDED_CYCLE_DATE
+if [[ $cdate_crnt_fhr -ge ${FIRST_BLENDED_CYCLE_DATE} ]]; then
+if [[ $DO_BLENDING == "TRUE" && $EXTRN_MDL_NAME_ICS == "GDASENKF" ]]; then
```

Comment:
Logically, lines 721-726 is expected to be put inside the `if DO_BLENDING` block.

Reply:
Also, it looks like lines 726 and 727 can be combined into one line.

Reply:
done